lib/Behavior/Audit/NonLocalizedTimestampBehavior.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\DocumentManager\Behavior\Audit;
+
+/**
+ * Adds the date when created and lastly changed the document (without locale).
+ */
+interface NonLocalizedTimestampBehavior extends TimestampBehavior
```

Comment:
Would prefer this to be called `TimestampBehavior` and the other interface should be the `LocalizedTimestampBehavior`. Negations in class names make things more complicated to understand... I think we don't have a BC promise here, do we? Then I would definitely rename this, otherwise we'll have this debt for a long time...